frontend/src/components/TicketTypes/AdminTicket.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import AdminTicket from './AdminTicket';
+import renderer from 'react-test-renderer';
+
+it('renders correctly', () => {
+  const tree = renderer.create(<AdminTicket />).toJSON();
```

Comment:
@LLaszlo89 you should have passed a title, a description and a valid iconName property for the component